site/gatsby-site/src/components/incidents/IncidentIdField.js

```diff
@@ -11,6 +11,7 @@ export default function IncidentIdField({
   placeHolder = '',
   className = '',
   showIncidentData = true,
+  required = false,
```

Comment:
I would also add validation in this component if this field is required (because as it is this property is only cosmetic), but it could be done in another PR

Reply:
Agreed. The Incident ID field validation is currently handled by the yup schema.
There are two forms that are using the `<IncidentIdField>` component:

- `IncidentReportForm.js` with the Incident Id field required
```
incident_id: yup.number().positive().integer('*Must be an incident number').required()
```
- `SubmissionForm.js` with the Incident Id field not required
```
incident_id: yup.number().positive().integer('*Must be an incident number or empty')
```